File: 1876-substrings-of-size-three-with-distinct-characters/1876-substrings-of-size-three-with-distinct-characters.cpp
class Solution {
public:
    
    int countGoodSubstrings(string s) {
         int len = s.length();
      int cnt = 0;
        if(len < 3)
            
           
            return 0;
        for(int i=0; i<=len-3; i++){
            if(s[i] != s[i+1] && s[i] != s[i+2] && s[i+1] != s[i+2])
                cnt++;
        }
        return cnt;
    }
};

File: 1876-substrings-of-size-three-with-distinct-characters/1876-substrings-of-size-three-with-distinct-characters.java
class Solution {
    public int countGoodSubstrings(String s) {
        int result = 0;
	for(int i=1;i<s.length()-1;i++)
		if(s.charAt(i-1) != s.charAt(i) && s.charAt(i+1) != s.charAt(i) && s.charAt(i+1) != s.charAt(i-1))
			result++;
	return result;
    }
}